ArchiSteamFarm/Statistics.cs

```diff
@@ -102,8 +102,8 @@ internal sealed class Statistics : IDisposable {
 				}
 
 				// Don't announce if we don't meet conditions
-				string tradeToken;
-				if (!await ShouldAnnounce().ConfigureAwait(false) || string.IsNullOrEmpty(tradeToken = await Bot.ArchiWebHandler.GetTradeToken().ConfigureAwait(false))) {
+				string tradeToken = await Bot.ArchiHandler.GetTradeToken().ConfigureAwait(false);
```

Comment:
This will ask for token before it's deemed as needed, previous way asked only if `ShouldAnnounce()`.